diff -urN xffm-gui-4.5.0/deskview/Makefile.am xffm-gui-4.5.0.new/deskview/Makefile.am
--- xffm-gui-4.5.0/deskview/Makefile.am	2006-05-23 23:10:44.000000000 +0200
+++ xffm-gui-4.5.0.new/deskview/Makefile.am	2007-02-06 00:26:45.000000000 +0100
@@ -34,6 +34,7 @@
 
 libxffm_deskview_la_LDFLAGS = $(COMMON_LDFLAGS) 
 
+libxffm_deskview_la_LIBADD = $(LIBXFFM_LIBS)
 
 common_FLAGS = 								\
 	-DLIBDIR=\"$(libdir)\"						\
diff -urN xffm-gui-4.5.0/iconview/Makefile.am xffm-gui-4.5.0.new/iconview/Makefile.am
--- xffm-gui-4.5.0/iconview/Makefile.am	2006-05-20 03:35:06.000000000 +0200
+++ xffm-gui-4.5.0.new/iconview/Makefile.am	2007-02-06 00:25:46.000000000 +0100
@@ -40,6 +40,7 @@
 
 libxffm_gridview_la_LDFLAGS = $(COMMON_LDFLAGS) 
 
+libxffm_gridview_la_LIBADD = $(LIBXFFM_LIBS)
 
 common_FLAGS = 								\
 	-DLIBDIR=\"$(libdir)\"						\
diff -urN xffm-gui-4.5.0/treeview/Makefile.am xffm-gui-4.5.0.new/treeview/Makefile.am
--- xffm-gui-4.5.0/treeview/Makefile.am	2006-05-20 03:35:09.000000000 +0200
+++ xffm-gui-4.5.0.new/treeview/Makefile.am	2007-02-06 00:18:47.000000000 +0100
@@ -50,6 +50,9 @@
 
 libxffm_treeview_la_LDFLAGS = $(COMMON_LDFLAGS) 
 
+libxffm_treeview_la_LIBADD = \
+	$(LIBXFFM_LIBS)							\
+	$(LIBXML_LIBS)
 
 common_FLAGS = 								\
 	-DLIBDIR=\"$(libdir)\"						\
